x/staking/spec/05_end_block.md

```diff
@@ -75,3 +76,11 @@ Complete the unbonding of all mature `Redelegation.Entries` within the
 - remove the mature entry from `Redelegation.Entries`
 - remove the `Redelegation` object from the store if there are no
   remaining entries.
+
+## Putting unbonding operations on hold
```

Comment:
what is the use case for this?

Reply:
see https://github.com/cosmos/cosmos-sdk/pull/12830#discussion_r943424722